changelog/963.feature.rst

```diff
@@ -0,0 +1 @@
+Further developed grid objects, improved documentation.
```

Comment:
This should be a little more descriptive.  At least mention that you're adding the Jupyter notebooks in this PR.

Reply:
I've updated the changelog to mention the notebooks. I could add more (akin to what's listed in the text at the top of this PR), but that will end up being a pretty long changelog entry since the changes were both broad and deep in scope.